src/main/java/net/openhft/chronicle/queue/impl/single/StoreAppender.java

```diff
@@ -530,7 +530,6 @@ public void writeBytes(@NotNull final BytesStore bytes) throws UnrecoverableTime
      */
     public void writeBytes(final long index, @NotNull final BytesStore bytes) {
         throwExceptionIfClosed();
-        checkAppendLock();
         writeLock.lock();
```

Comment:
removed as writeBytesInternal calls. Or did you want to lock in case a non-sink calls this?

Reply:
actually I will put back

Reply:
Ignore this comment